src/main/java/md/jack/task8/MathUtils.java

```diff
@@ -0,0 +1,30 @@
+package md.jack.task8;
+
+import java.util.Random;
+import java.util.stream.DoubleStream;
+import java.util.stream.Stream;
+
+class MathUtils
```

Comment:
Utils class has to be uninstantiable and unable to extended by any other class

- Declare Utils classes as final
- create private empty constructor